.github/workflows/ci.yml

```
@@ -10,41 +10,21 @@ on:
-
-      - uses: cvmfs-contrib/github-action-cvmfs@v4
-
-      - uses: aidasoft/run-lcg-view@v4
-        with:
-          release-platform: "${{matrix.lcg}}/${{env.platform}}"
-          run: |
-            set -e
-            echo "::group::Set-up key4hep"
-            source setup.sh
+    runs-on: ubuntu-latest
+    container:
```

actionlint (1.7.4)

string should not be empty

(syntax-check)